src/main/java/org/ajoberstar/gradle/stutter/StutterPlugin.java

```diff
@@ -15,13 +15,13 @@
  */
 package org.ajoberstar.gradle.stutter;
 
-import org.gradle.api.Project;
 import org.gradle.api.Plugin;
+import org.gradle.api.Project;
 import org.gradle.api.Task;
-import org.gradle.plugin.devel.GradlePluginDevelopmentExtension;
-import org.gradle.api.tasks.testing.Test;
-import org.gradle.api.tasks.SourceSet;
 import org.gradle.api.plugins.JavaPluginConvention;
+import org.gradle.api.tasks.SourceSet;
+import org.gradle.api.tasks.testing.Test;
+import org.gradle.plugin.devel.GradlePluginDevelopmentExtension;
 
 public class StutterPlugin implements Plugin<Project> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public class by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=squid%3AUndocumentedApi)